components/Navbar.js

```diff
@@ -69,7 +77,9 @@ const Navbar = ({ border, links, title, heart, back, empty, colour }) => {
       colour={colour}
       border={border}
     >
-      {back && <Back />}
+      {back && (
+        <Back onClick={() => (backToHome ? Router.push('/') : Router.back())} />
```

Comment:
Could you pass backToHome to the styled component and use the onClick there? Just for consistency.